src/Cassandra.IntegrationTests/Core/ClientTimeoutTests.cs

```diff
@@ -53,16 +53,16 @@ public void Should_Move_To_Next_Host_For_Simple_Queries()
             {
                 var session = cluster.Connect();
                 //warmup
-                TestHelper.Invoke(() => session.Execute("SELECT key FROM system.local"), 10);
+                TestHelper.Invoke(() => session.Execute("SELECT key FROM system.local WHERE key='local'"), 10);
```

Comment:
It's used so many times in the same file, perhaps we can create a constant string for it?

Reply:
private const string IdleQuery = "SELECT key FROM system.local WHERE key='local'";